docs/system/hardware-metrics.md

```diff
@@ -93,11 +93,11 @@ Examples: physical server, switch or disk array.
 **Description:** A battery in a computer system or an UPS.
 
 | Name                      | Description                                                                   | Units | Instrument Type ([*](/docs/general/metrics.md#instrument-types)) | Value Type | Attribute Key(s)                                                            | Attribute Values                                      |
-| ------------------------- | ----------------------------------------------------------------------------- | ----- | ------------------------------------------------- | ---------- | --------------------------------------------------------------------------- | ----------------------------------------------------- |
+|---------------------------| ----------------------------------------------------------------------------- | ----- | ------------------------------------------------- | ---------- | --------------------------------------------------------------------------- | ----------------------------------------------------- |
```

Comment:
Not sure this change is made on purpose (other columns don't follow the same pattern), and same for other similar changes in table formatting.